### ISSUE TITLE
documentation on how to configure credentials from the appsettings.json file

### ISSUE THREAD

**Issue by doreille** [serilog/serilog-sinks-email#136]
The configuration parameters have changed in version 3.0.
Prior to version 3.0, SMTP credentials were configured using the userName and password attributes.
From version 3.0 onwards, the SMTP credentials are configured using the credentials attribute which has the ICredentialsByHost interface type.

I haven't found any documentation or examples of how to set this. I try

 "credentials": {
   "type": "System.Net.NetworkCredential, System.Net.Primitives",
   "UserName": ""#########"#",
   "Password": "#########"
 }

but this return the error System.InvalidOperationException: 'Cannot create instance of type 'System.Net.ICredentialsByHost' because it is either abstract or an interface.'

**Comment by jlrouzies-mantu** [comment]
I agree Serilog documentations are lack luster when it comes to appsetting.json.

I am trying to fill `"ConnectionSecurity": "StartTls",` and I just get String error, and don't even know what to use instead.

**Comment by bartelink** [comment]
As evidenced by the three week lag and the noise of a "me too" on a duplicate, [better to ask this on SO](https://github.com/serilog/serilog-sinks-email/issues/138#issuecomment-2276202764)

**Comment by alwilton** [comment]
Perahps this will help
[.Net Core and Serilog Email sink - JSON Config](https://stackoverflow.com/a/59914193/3321499)

**Comment by Trapulo** [comment]
same problem here

**Comment by w3ori** [comment]
How to set `connectionSecurity: MailKit.Security.SecureSocketOptions.None` in appsettings?

**Comment by akordowski** [comment]
I am also looking for an documentation
